tasks/src/tabular-regression/data.ts

```diff
@@ -0,0 +1,59 @@
+import type { TaskData } from "../Types";
+
+import { PIPELINE_DATA } from "../../../js/src/lib/interfaces/Types";
+import { TASKS_MODEL_LIBRARIES } from "../const";
+
+const taskData: TaskData = {
+	datasets: [
+		{
+			description: "Dataset containing tip amounts for food servers.",
+			id:          "scikit-learn/tips",
+		},
+		{
+			description: "Dataset containing mile-per-gallon (mpg) information for various vehicles",
+			id:          "scikit-learn/auto-mpg",
+		},
+		{
+			description: "Dataset containing measurements of fish at a market.",
```

Comment:
I think the above two are enough 🙂 WDYT?

Reply:
Yes agreed. I will drop one!